Source: https://github.com/mesonbuild/meson/issues/1736
Upstream: Unknown
Reason: prevents meson from caching a variable with the value of the cross-compile pkg-config that can be later used to fetch the value of a native pkg-config, fixes problems with finding native: true dependencies.

--- mesonbuild/dependencies/base.py
+++ mesonbuild/dependencies/base.py
@@ -448,7 +448,6 @@ class PkgConfigDependency(ExternalDependency):
                 potential_pkgbin = ExternalProgram.from_cross_info(environment.cross_info, 'pkgconfig')
                 if potential_pkgbin.found():
                     self.pkgbin = potential_pkgbin
-                    PkgConfigDependency.class_pkgbin = self.pkgbin
                 else:
                     mlog.debug('Cross pkg-config %s not found.' % potential_pkgbin.name)
         # Only search for the native pkg-config the first time and
<Paste>
